Meadowlark-js/backends/meadowlark-postgresql-backend/test/integration/Delete.test.ts

```diff
@@ -51,8 +52,8 @@ const newGetRequest = (): GetRequest => ({
 });
 
 describe('given the delete of a non-existent document', () => {
-  let client;
-  let deleteResult;
+  let client: PoolClient;
+  let deleteResult: DeleteResult;
```

Comment:
Unfortunately, you followed my mistake of not putting types on these. 🤣  I added them, which led me to a bug in your implementation code. Don't forget to add the types when you un-comment my code for reference testing!